sysdeps/unix/sysv/linux/i386/clone.c

```diff
@@ -1,4 +1,53 @@
-int __GI___clone (void) {
-  return 0;
+#include <sched.h>
+#include "libioP.h"
```

Comment:
this file needs a bunch of comments

Reply:
This is no longer useful anymore, I will restore to the original one